stock_quantity_history_location/static/src/components/inventory_report/inventory_report.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template" xml:space="preserve">
+    <t t-inherit="stock.InventoryReport.Buttons" t-inherit-mode="extension" owl="1">
+        <xpath expr="(//div/*)[last()]" position="after">
+            <t t-if="this.multi_location" name="has_multi_location">
+                <button
+                    t-if="
+                        props.resModel == 'stock.quant'
+                        and (
+                            !props.context.inventory_mode
+                            or props.context.inventory_report_mode
+                        )
+                        and !props.context.no_at_date
```

Comment:
Add an ID to this button, so if we (or anybody else) migrate the module `stock_account_quantity_history_location`, xpaths are simpler